### PR TITLE
Feat/outlook rooms addin

### PR DESCRIPTION
**Pull request by juliejgiles**
**Description of the change**

- Added `function-file.ts` and `function-file.html` to assets folder for builds.
- Updated `manifest.xml` to point to function-files
- Updated `manifest.xml` to point to deployed url 
- Added PlaceOS icons in `manifest.xml` instead of placeholder icons.
- Added extra button and viewing pane to `manifest.xml` for upcoming bookings
- Started on Upcoming Bookings component

**Benefits**
`manifest.xml` had incorrect references

**Possible drawbacks**
N/A

**Applicable issues**
N/A

**Additional information**
N/A

### CODE REVIEW

**Comment by MrYuion** [comment]
What are `function-file.ts` and `function-file.html` for?

**Comment by juliejgiles** [comment]
> What are `function-file.ts` and `function-file.html` for?

Hi Alex, 

I believe that `function-file.ts` imports and initializes Office, which must be run at least once to load the taskpane. The `function-file.html` is an empty UI template that calls the `function-file.ts`, and is referenced in line 97 of the `manifest.xml` file for clicking on the add-in button to open up the taskpane.

`function-file.ts` and `function-file.html` were previously referenced in the `manifest.xml` file but I am trying to change it to the deployed url path.

**Comment by MrYuion** [comment]
Hmm... I don't see those files in the PR.
Also shouldn't `function-file.ts` be a javascript file instead?
It doesn't seem to be compiled so there might be issues running it as a `.ts` file.

**Comment by juliejgiles** [comment]
> Hmm... I don't see those files in the PR. Also shouldn't `function-file.ts` be a javascript file instead? It doesn't seem to be compiled so there might be issues running it as a `.ts` file.

The function-files don't form a part of this PR because they had been approved in PR #120. The `manifest.xml` file was referencing them in a relative path which needs updating.
image

Good point about the `.ts` file - I'll update this.

File: apps/outlook-rooms-addin/manifest.xml [hunk 1]
```diff
@@ -79,21 +89,26 @@
...
       </bt:Images>
       <bt:Urls>
-        <bt:Url id="functionFile" DefaultValue="https://127.0.0.1:3000/function-file/function-file.html"/>
+      <bt:Url id="functionFile" DefaultValue="https://microsoft.poc.placeos.com/outlook/function-file/function-file.html"/> 
+        <bt:Url id="Taskpane.Url" DefaultValue="https://microsoft.poc.placeos.com/outlook/"/>
+        <bt:Url id="BookingsButton.Url" DefaultValue="https://microsoft.poc.placeos.com/outlook/upcoming"/>
+        <!-- <bt:Url id="functionFile" DefaultValue="https://127.0.0.1:3000/function-file/function-file.html"/>
         <bt:Url id="Taskpane.Url" DefaultValue="https://127.0.0.1:4200/"/>
-        <!-- <bt:Url id="Taskpane.Url" DefaultValue="https://portal-dev.placeos.run/#/"/> -->
+        <bt:Url id="BookingsButton.Url" DefaultValue="https://127.0.0.1:4200/upcoming"/> -->
```
Comment (MrYuion) [inline apps/outlook-rooms-addin/manifest.xml]: Should this be 127.0.0.1?

Comment (juliejgiles) [reply apps/outlook-rooms-addin/manifest.xml]: > Should this be 127.0.0.1?

Oops, lines 100-102 should have been commented out, as they are outdated versions of lines 97-99 here. I'll remove these.

File: apps/outlook-rooms-addin/src/app/rooms/map.service.ts [hunk 2]
```diff
@@ -17,17 +18,44 @@ export interface Locatable {
     zones?: string[];
 }
 
+export interface mapsList {
```
Comment (MrYuion) [inline apps/outlook-rooms-addin/src/app/rooms/map.service.ts]: Types should be in PascalCase.
e.g. `ClassName`

**Review by MrYuion — COMMENTED** [review]
Looks good. There are a few formatting and consistency issues with variable and method names but is otherwise fine.

**Pull request merged**